dali/kernels/imgproc/resample/resampling_impl_cpu.h

```diff
@@ -69,7 +66,68 @@ void ResampleCol(Out *out, const In *in, int x, int w, const int32_t *in_columns
     }
 
     for (int c = 0; c < channels; c++)
-      out[channels * x + c] = clamp<Out>(tmp[c]);
+      out[channels * x + c] = ConvertSat<Out>(tmp[c]);
+  }
+}
+
+inline bool GetFirstAndLastRegularCol(int &first_regular_col,
+                                      int &last_regular_col,
+                                      int out_width, int in_width, const int *in_col_idxs,
+                                      int support) {
+  bool flipped = in_col_idxs[out_width-1] < in_col_idxs[0];
+  first_regular_col = 0;
+  last_regular_col = out_width - 1;
+  if (flipped) {
+    while (first_regular_col < out_width && in_col_idxs[first_regular_col] + support > in_width)
+      first_regular_col++;
+    while (last_regular_col >= 0 && in_col_idxs[last_regular_col] < 0)
```

Comment:
It's the flipped case - let's assume that out_width is 5, in_width is 4 and support is 4.
```
in_col_idxs = {
  3, 2, 0, -1, -2  // decreasing, because it's flipped
};
```
In this case last_regular_col will be 2 and first regular_col will be 3. In this particular case they intersect (last < first), which means that there's no regular column.